src/shared/middleware/make-request-middleware.js

```diff
@@ -14,14 +14,11 @@ export const makeRequestMiddleware = <TReq: $Request, TRes: $Response>(
 ): Promise<Middleware<TReq, TRes>> => {
     if (mode === "production") {
         /**
-         * In production, we're using the Google middleware. This adds the
-         * log property to the request, allowing us to associate log entries
-         * with a request trace, if the request is being traced.
-         *
-         * WORKAROUND: We must pass the corresponding transport here or it will
-         * get added for us and then we'll have double logging.
+         * In production, we're using the Google logging-winston middleware.
+         * This adds the log property to the request, allowing us to associate
+         * log entries with a request trace, if the request is being traced.
          */
-        return lw.express.makeMiddleware(logger, new lw.LoggingWinston());
```

Comment:
The change to the `makeMiddleware` function in `src/middleware/express.ts` as part of https://github.com/googleapis/nodejs-logging-winston/pull/553 means that the `LoggingWinston` transport that we add when we create our logger is now properly found and accepted. This means that we don't need to pass one here to trick the middleware into not adding one, and in fact, passing one here now causes issues.

Reply:
Do we have to upgrade `@google-cloud/logging-winston` in package.json?

Reply:
Dependabot already did.